mandatory/2-function-creation.js

```diff
@@ -16,7 +27,17 @@ Write a function that:
 - numbers greater 100 must be replaced with 100
 */
 
-function formatPercentage(num) {}
+function formatPercentage(num) {
+  if (num >= 100){
+    return "100%"
+  }
+
+  else {
+    return num = parseFloat(num.toFixed(2)) +"%";
```

Comment:
I am seeing a variable assignment and return statement at the same time. Can you fix this?